src/SGEMM/CuTropicalSGEMM_example.jl

```diff
@@ -0,0 +1,46 @@
+using CUDA, Test, BenchmarkTools, Dates
```

Comment:
The final form of this project will be a package, then we should following the convention of Julia package system.
1. The `src` folder must only contain a package source code, the tests should be in the `test` folder.
2. Benchmarks must be in a different folder.
3. The `so` file should be an artifact, please check: https://www.juliafordatascience.com/artifacts/

Reply:
For C-wrappers, there are a lot of references, such as `KaHyPar`: https://github.com/kahypar/KaHyPar.jl/tree/master/src